lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,79 @@
+# Gerrit
+
+## Supported Gerrit Versions
+
+All Gerrit 3.x Versions should be working.
```

Comment:
Do you really intend for us to support `3.x` versions that Gerrit themselves have end-of-lifed?

I would expect us to match the support policy of upstream Gerrit? That makes our lives easier, as we don't have to support old stuff.

Reply:
I personally do not provide any support ;)

If you want to match with the Gerrit support policy then you can do this.
I just wanted to show which versions it should work with.

Maybe you should write this section yourself.

Reply:
> I personally do not provide any support ;)

Haha. 😄

> If you want to match with the Gerrit support policy then you can do this. I just wanted to show which versions it should work with.

If we say works with `2.x` readers of the docs will "expect" support for `2.x`, even if upstream Gerrit does not anymore. So we should only list things that we want to support.

@rarkins @viceice what kind of support policy do you guys want to follow for the new Gerrit platform support? The Gerrit maintainers support the _last three_ versions.

> Maybe you should write this section yourself.

I'll do that, once I know the maintainers policy. 😉

Reply:
yes, we normally only support maintained platform versions. but we support older versions with best effort, that mostly needs community contribution as we don't have such old platforms to test.

Reply:
```suggestion
Renovate supports all Gerrit 3.x versions.
```

Let's rewrite a bit.